frontend/src/wrapper/EntrySearch.tsx

```diff
@@ -53,28 +55,34 @@ export default function EntrySearch() {
   const modelList = useMemo(
     () =>
       entries.map((entry) => (
-        <Box key={entry.id} sx={{ maxWidth: '300px' }}>
+        <Box key={entry.id} sx={{ maxWidth: '400px' }}>
           <Link href={`/${entry.kind}/${entry.id}`} noLinkStyle>
-            <ListItemButton>
-              <ListItemText
-                primary={entry.name}
-                secondary={entry.description}
-                slotProps={{
-                  primary: {
-                    style: {
-                      whiteSpace: 'nowrap',
-                      textOverflow: 'ellipsis',
-                      overflow: 'hidden',
-                      color: theme.palette.primary.main,
+            <ListItem
+              disablePadding
+              secondaryAction={<Chip label={toTitleCase(EntryKindLabel[entry.kind])} size='small' />}
+            >
+              <ListItemButton>
+                <ListItemText
+                  primary={entry.name}
+                  secondary={entry.description}
+                  slotProps={{
+                    primary: {
+                      style: {
+                        whiteSpace: 'nowrap',
+                        textOverflow: 'ellipsis',
+                        overflow: 'hidden',
+                        color: theme.palette.primary.main,
+                        maxWidth: '400px',
+                      },
                     },
-                  },
 
-                  secondary: {
-                    style: { whiteSpace: 'nowrap', textOverflow: 'ellipsis', overflow: 'hidden' },
-                  },
-                }}
-              />
-            </ListItemButton>
+                    secondary: {
+                      style: { whiteSpace: 'nowrap', textOverflow: 'ellipsis', overflow: 'hidden', maxWidth: '400px' },
```

Comment:
We can use `sx` here instead of `style`.

Reply:
Also, see my other comment about changing the `maxWidth` property to be margin instead